docs_input/api/casting/as_complex_float.rst

```diff
@@ -0,0 +1,24 @@
+.. _as_complex_float_func:
+
+as_complex_float
+=================
+
+Cast an operator to cuda::std::complex<float>
+
+.. doxygenfunction:: matx::as_complex_float(T t)
```

Comment:
Do we need these for the half types?

Reply:
Probably, but I'm not sure what the best types would be. matxHalfComplex<matxFp16> and matxHalfComplex<matxBf16>? The cast operator name would need to differentiate between matxFp16 and matxBf16, or pick only one as the return value for as_complex_half.

Reply:
Sure, let's leave it for now and add if someone requests it.